pkgs/by-name/en/ente-cli/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  ...
+}:
+let
+  pname = "ente-cli";
+  version = "0.2.1";
+
+in
+buildGoModule {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "ente-io";
+    repo = "ente";
+    rev = "refs/tags/cli-v${version}";
+    hash = "sha256-gIDJUj2pn8rndXWN69bZdVfRLVB7AybXHMcioG2NI1k=";
+    fetchSubmodules = false;
+    sparseCheckout = [ "cli" ];
+  };
+
+  modRoot = "./cli";
+
+  vendorHash = "sha256-Gg1mifMVt6Ma8yQ/t0R5nf6NXbzLZBpuZrYsW48p0mw=";
+
+  CGO_ENABLED = 0;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.AppVersion=cli-v${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/{cli,${pname}}
+  '';
+
+  passthru.updateScript = nix-update-script {
+    extraArgs = [
+      "--version-regex"
+      "cli-(.+)"
+    ];
+  };
+
+  meta = {
+    description = "CLI client for downloading your data from Ente.";
```

```suggestion
    description = "CLI client for downloading your data from Ente";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes